doc/advanced_setup.rst

```diff
@@ -103,6 +103,29 @@ runtime accordingly.
 If you use another Python setup and you encounter some difficulties please
 report them on the MNE mailing list or on github to get assistance.
 
+Installing Mayavi
+^^^^^^^^^^^^^^^^^
+
+If you have Anaconda installed (recommended), the easiest way to install `mayavi` is to do::
```

Comment:
The first line sentence should probably mention that Mayavi is only available for Python2.7, so if you're running 3.3+, you can't install it